src/Silex/Provider/RememberMeServiceProvider.php

```diff
@@ -19,6 +19,9 @@
 use Symfony\Component\Security\Http\RememberMe\ResponseListener;
 
 /**
+ * @deprecated Use `\Spryker\Yves\Security\Plugin\Security\RememberMeSecurityPlugin` instead.
```

Comment:
We should then use simple message type in deprecated tag, since it cant beclicked and duplicating it only increases readability issues:

    @deprecated Use `RememberMeSecurityPlugin` instead.
    @see \Spryker\Yves\Security\Plugin\Security\RememberMeSecurityPlugin

Then it makes sense as is.

Reply:
I think we should still have both using FQCN as `@deprecation` could e.g. only be visible in a log file. 

I added a new line for readability. What do you think about?

Reply:
Then you would check the code for the details. We must not use duplication here if we move forward with this new change.